toolkits/global/packages/global-corporate-footer/README.md

```diff
@@ -48,10 +48,10 @@ CMP users don't need to link to a cookie policy from the footer as it's covered
 
 See an example implementation in the [`demo` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/demo).
 
-For One Trust, the generated HTML of the Manage cookies / Do not sell data interactive element looks like this:
+For One Trust, the generated HTML of the "Your privacy choices/Manage cookies" interactive element looks like this:
 
 ```html
-<button onclick="Optanon.ToggleInfoDisplay()" class="c-corporate-footer__link">Manage cookies / Do not sell my data</button>
+<button onclick="Optanon.ToggleInfoDisplay()" class="c-corporate-footer__link">Your privacy choices/Manage cookies</button>
```

Comment:
```suggestion
<button class="c-corporate-footer__link" data-cc-action="preferences">Your privacy choices/Manage cookies</button>
```

Reply:
Sorry, I just noticed this. Should update the example to use our cookie consent rather than OneTrust's.

Reply:
Do you know where I can see how is the most updated way to do it?

Reply:
https://cookie-consent.public.springernature.app/docs/getting-started/installation